col.go

```diff
@@ -91,6 +91,46 @@ func (f *File) SetColVisible(sheet, col string, visible bool) error {
 	return err
 }
 
+// SetColsVisibleFrom provides a function to set visible of all the columns
+// after a given one in a worksheet. For example, hide column D in Sheet1:
+//
+//    err := f.SetColsVisibleFrom("Sheet1", "D", false)
+//
+func (f *File) SetColsVisibleFrom(sheet, col string, visible bool) error {
```

Comment:
I propose that modify the exist function `SetColVisible` like `SetColStyle`:

```go
func SetColVisible(sheet, columns string, visible bool) error
```

to make it accept the specified column range:

```go
SetColVisible("Sheet1", "F", false)
SetColVisible("Sheet1", "F:XFD", false)
```

Reply:
Ok, i'll post that soon.

Reply:
Please see PR #565